src/session-adapters/cookieAdapter.ts

```diff
@@ -19,33 +20,44 @@ const createScopedKey = ({
 // We do not use `clientId` in cookie adapter,
 // because different plugins will have different domain names,
 // and it's enough to differentiate these cookie values.
+
+const sessionKey = 'sb.auth'
+//NOTE: possibly cookieAdapter can become createCookieAdapter(key: string)
 export const cookieAdapter: Adapter = {
-  getItem: ({ req, spaceId, userId, key }) => {
-    const cookie = getCookie(req, createScopedKey({ spaceId, userId, key }))
+  getItem: ({ req, spaceId, userId }) => {
```

Comment:
I would even go as far as to rename these functions to include session (e.g.: setSession, getSessions, etc.) to make it more specific.

Reply:
Yeah I agree.